includes/admin/load.php

```diff
@@ -330,8 +330,10 @@ static function ( $name ) {
 	}
 
 	if ( isset( $_GET['activate'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$plugin_settings_link = perflab_get_plugin_settings_link( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		wp_admin_notice(
-			esc_html__( 'Feature activated.', 'performance-lab' ),
+			/* translators: %s: Settings link of the plugin */
+			sprintf( esc_html__( 'Feature activated. Review %s.', 'performance-lab' ), $plugin_settings_link ),
```

Comment:
Oh wait, there's a more _important_ point here. Many of the plugins don't have settings links. So this needs to be made conditional. Also, ideally "Settings" would rather be "settings". So for example:

```php
$message = __( 'Feature activated.', 'performance-lab' );

$plugin_settings_link = perflab_get_plugin_settings_link( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
if ( $plugin_settings_link ) {
	$p = new WP_HTML_Tag_Processor( $plugin_settings_link );
	if ( $p->next_tag( array( 'tag_name' => 'A' ) ) && $p->get_attribute( 'href' ) ) {
		/* translators: %s is the settings URL */
		$message .= ' ' . sprintf( __( 'Review <a href="%s">settings</a>.', 'performance-lab' ), esc_url( $p->get_attribute( 'href' ) ) )
	}
}
wp_admin_notice(
	wp_kses( $message, array( 'a' => array( 'href' => array() ) ),
	array(
		'type'        => 'success',
		'dismissible' => true,
	)
);
```